Go/README.md

```diff
@@ -78,47 +75,51 @@ You can then use this package in your Go programs like this:
 package main
 
 import (
-    "net/http"
-    "log"
     "github.com/gorilla/mux" // Your imported package
+    "net/http"
 )
 
-func YourHandler(w http.ResponseWriter, r *http.Request) {
+func yourHandler(w http.ResponseWriter, r *http.Request) {
     w.Write([]byte("Gorilla!\n"))
 }
 
 func main() {
     r := mux.NewRouter()
     // Routes consist of a path and a handler function.
-    r.HandleFunc("/", YourHandler)
+    r.HandleFunc("/", yourHandler)
 
     // Bind to a port and pass our router in
-    log.Fatal(http.ListenAndServe(":8000", r))
+    panic(http.ListenAndServe(":8000", r))
 }
 ```
 
-## Format your code
+## Tooling and learning
+
+### Format your code
 
 Go has a built-in tool that automatically formats Go source code.
 
 To format a single file run:
 
-    $ gofmt -w yourcode.go
+    gofmt -w yourcode.go
+
+You can also format an entire package:
 
-You can also format an entire package (**Note that the command is different from formatting a single file**):
+    go fmt path/to/your/package
```

Comment:
Here we say `go fmt` but above we say `gofmt`, I'm guessing this is the correct one?

Reply:
Ah didn't see the Note below. Ignore me.